<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneBook</class>
 <widget class="QMainWindow" name="PhoneBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WHI3PER::PhoneBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {  
	font: 11pt &quot;Comic Sans MS&quot;;
     background-color: rgb(25, 25, 25);
	 color: rgb(255, 255, 0);
}

QTextBrowser {
	font: 11pt &quot;Consolas&quot;; 
	background-color: rgb(50, 50, 50);  
	color: rgb(0, 255, 255); 
}

QGroupBox {  color: rgb(85, 255, 127); }
QPushButton { background-color: rgb(55, 55, 55); }
QComboBox {  background-color: rgb(55, 55, 55);  color: rgb(85, 255, 127); }
QMenuBar { color: rgb(85, 255, 127); }
QTabBar::tab { background-color: rgb(55, 55, 55); }
QLineEdit { background-color: rgb(50, 50, 50); }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>306</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Name</string>
      </property>
      <widget class="QLineEdit" name="NAME">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Phone's</string>
      </property>
      <widget class="QLineEdit" name="PHONE1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>XXXX XXX XXXX</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PHONE2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>XXXX XXX XXXX</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Gamil</string>
      </property>
      <widget class="QLineEdit" name="GMAIL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="ADD_BUTTON">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>340</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Contact</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Remove</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Name</string>
      </property>
      <widget class="QLineEdit" name="SEARCH1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter a name for remove</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="REMOVE_BUTTON">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Contact</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Edit</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Name</string>
      </property>
      <widget class="QLineEdit" name="SEARCH2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter a name for search</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>New Name</string>
      </property>
      <widget class="QLineEdit" name="NEW_NAME">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>New Phone's</string>
      </property>
      <widget class="QLineEdit" name="NEW_PHONE1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>XXXX XXX XXXX</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NEW_PHONE2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>XXXX XXX XXXX</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>New Gamil</string>
      </property>
      <widget class="QLineEdit" name="NEW_GMAIL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="EDIT_BUTTON">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit Contact</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>List</string>
     </attribute>
     <widget class="QPushButton" name="SHOW_ALL">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All Contact's</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SHOW_NAMES">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All Name's</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SHOW_PHONES">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All Phone's</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SHOW_GMAILS">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All Gamil's</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Name</string>
      </property>
      <widget class="QLineEdit" name="SEARCH3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Enter a name for search</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="SEARCH_BUTTON">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search Contact</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>10</y>
      <width>306</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QTextBrowser" name="LOG">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>286</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="EXIT">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>450</y>
      <width>86</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ABOUT">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="ABOUT_2">
   <property name="text">
    <string>About Me</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
